docs/references/astro/overview.mdx

```diff
@@ -23,7 +23,9 @@ Clerk Astro provides a set of useful [stores](https://github.com/nanostores/nano
 - [`$sessionListStore`](/docs/references/astro/session-list-store)
 - [`$organizationStore`](/docs/references/astro/organization-store)
 
-## General Reference
+## Server-side helpers
```

Comment:
I think the locals need to be added here, or somewhere on the page. Astro is on our list of going thru those docs and editing/validating them, but while we're here, we should do this right. we can pair on it if you want

Reply:
https://github.com/clerk/clerk-docs/pull/1815/commits/6ec92b4ca76dad4aaac706ed0bc4d5809524abf3